src/plugins/sql-support/plugin.js

```diff
@@ -389,9 +389,14 @@ QueryBuilder.extend({
                 var opVal = sqlrl.call(this, value, data.operation);
                 if (operator == 'NOT LIKE') opVal.op = 'not_' + opVal.op;
 
+                var left_value = data.left.value.value;
+                if (data.left.values) {
+                  left_value = data.left.values.join('.');
+                }
+
                 curr.rules.push({
-                    id: that.change('getSQLFieldID', data.left.value, value),
-                    field: data.left.value,
+                    id: that.change('getSQLFieldID', left_value),
```

Comment:
It does make sense, it can be used to discriminate when multiple filters work on the same field.